addons/web_editor/controllers/main.py

```diff
@@ -191,7 +191,8 @@ def add_data(self, name, data, is_image, quality=0, width=0, height=0, res_id=Fa
             try:
                 data = tools.image_process(data, size=(width, height), quality=quality, verify_resolution=True)
                 mimetype = guess_mimetype(b64decode(data))
-                if mimetype not in SUPPORTED_IMAGE_MIMETYPES:
+                img = ImageProcess(False, verify_resolution=False)
```

Comment:
I don't understand what this is supposed to do as it looks like we're not processing any image ? How could this call result in anything useful ?

Reply:
This call actually aids in determining whether the image is not an unsupported mimetype such as'svg', 'WEBP', 'riff' or another.
see: https://github.com/odoo/odoo/blob/e882bf4c1d30896984b73ca931ffa05e0f920414/odoo/tools/image.py#L78-L80 
and as a result, if any of these mimetype images are present, the "self.image" attribute will return "False.

Reply:
I must be missing some critical piece of understanding then because from my point of view this call does pass `False` as the value for `base64_source`, therefore `self.image` will always be `False` so the proposed changes in this PR makes no difference as to when an error is returned or not. What am I missing ?